src/new-client/src/app/site/[site]/create-availability/wizard/summary-step.tsx

```diff
@@ -0,0 +1,114 @@
+'use client';
+import NhsHeading from '@components/nhs-heading';
+import {
+  BackLink,
+  Button,
+  ButtonGroup,
+  Card,
+  SummaryList,
+  SummaryListItem,
+} from '@components/nhsuk-frontend';
+import { InjectedWizardProps } from '@components/wizard';
+import { parseAndValidateDateFromComponents } from '@services/timeService';
+import { useFormContext } from 'react-hook-form';
+import { AvailabilityTemplateFormValues } from './availability-template-wizard';
+
+const SingleOrRepeatingSessionStep = ({
+  goToPreviousStep,
+}: InjectedWizardProps) => {
+  const { getValues } = useFormContext<AvailabilityTemplateFormValues>();
+
+  const formValues = getValues();
+
+  const startDate = parseAndValidateDateFromComponents(
+    formValues.startDateDay,
+    formValues.startDateMonth,
+    formValues.startDateYear,
+  );
+
+  const endDate = parseAndValidateDateFromComponents(
+    formValues.endDateDay,
+    formValues.endDateMonth,
+    formValues.endDateYear,
+  );
+
+  const summary: SummaryListItem[] = [
```

Comment:
Should we remove the change actions given that they don't work. Can we do that?

Reply:
Yeah, I grumbled as I put them in. 
The only point I'd mention is that the [summary list component](https://service-manual.nhs.uk/design-system/components/summary-list) needs wrapping in a smaller container if the actions are missing as the line breaks extend to the right and I didn't want to introduce an extra element, but I'm happy to remove for now as I expect this all to change very soon